setup.py

```diff
@@ -340,6 +340,7 @@ def run(self):
     packages=packages,
     license="Apache License v2.0",
     include_package_data=True,
+    package_data={"onnx": ["py.typed", "*.pyi"]},
```

Comment:
One nit update: I removed adding `py.typed` in MANIFEST.in, because I think this line L343 will do the same thing here. I have tested the updated ONNX wheel can still produce correct type information.